Python Fonksiyon Çalışmaları.py

```diff
@@ -0,0 +1,96 @@
+import math
+from num2words import num2words
+
+""" İLK SORU """
+
+
+def bolunenSayiBulma(min_sayi, max_sayi, bolunecek_sayi):
+    list = []
+    for i in range(min_sayi, max_sayi + 1):
+        if i % bolunecek_sayi == 0:
+            list.append(i)
+
+    toplam_sayi = len(list)
+    print(list)
+    return toplam_sayi
+
+
+# aradaki_sayi = bolunenSayiBulma(23, 128, 9)
+# print(aradaki_sayi)
+
+"""İKİNCİ SORU"""

```

Comment:
burda mantık yanlış anlaşıldı sanırım, sayısının okuşunu çıktı olarak verilecek. ipucu olarak bunu bırakıyorum, 
`birler = ["", "Bir", "İki", "Üç", "Dört", "Beş", "Altı", "Yedi", "Sekiz", "Dokuz"]
    onlar = ["", "On", "Yirmi", "Otuz", "Kırk", "Elli", "Altmış", "Yetmiş", "Seksen", "Doksan"]`